### PR TITLE
Update Dash to version 3.0.2

### PR DESCRIPTION
**Pull request by gabeins**

### CODE REVIEW

File: Casks/dash.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
   version :latest
   sha256 :no_check
 
-  url 'https://kapeli.com/Dash.zip'
-  appcast 'https://kapeli.com/Dash.xml'
+  url "https://newyork.kapeli.com/downloads/v3/Dash.zip"
```
Comment (vitorgalvao) [inline Casks/dash.rb]: Double quotes are reserved for cases where string interpolation is needed. Don’t change them needlessly.

Comment (gabrielizaias) [inline Casks/dash.rb]: Sorry, I tested with interpolation using `version` on the string and changed before commiting.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by gabrielizaias** [comment]
Squashed :hammer:

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 97df350de01080c3ff42680181f2c63652d062f9 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**